src/main/java/racingcar/Racing.java

```diff
@@ -0,0 +1,61 @@
+package racingcar;
+
+import java.util.ArrayList;
+
+public class Racing {
+
+    private final int BOUND_RANDOM_NUMBER = 10;
+    private final int MORE_THEN_STANDARD_NUMBER = 4;
+
+    private Car[] racingCars;
+    private ArrayList<ArrayList<RacingRecord>> racingHistory = new ArrayList<>();
```

Comment:
```suggestion
    private List<List<RacingRecord>> racingHistory = new ArrayList<>();
```


인터페이스로 참조하면 어떠한 차이가 있을지 고민해보아요
`아이템 64. 객체는 인터페이스를 사용해 참조하라`를 참고해주세요 :)